bbb-meeting-api/src/main/resources/application.conf

```diff
@@ -0,0 +1,41 @@
+application = {
+    akka {
+      actor {
+        debug {
+          # enable DEBUG logging of all AutoReceiveMessages (Kill, PoisonPill et.c.)
+          autoreceive = on
+          # enable DEBUG logging of actor lifecycle changes
+          lifecycle = on
+        }
+      }
+
+      loggers = ["akka.event.slf4j.Slf4jLogger"]
+      loglevel = "DEBUG"
+  }
+
+  http = {
+    host = "0.0.0.0"
+    port = "8080"
```

Comment:
Maybe use 8082 for the port. I believe we have a tomcat instance running on 8080 already. At least I was running into an issue with Tomcat already running on port 8080. The recording API runs on 8081 so 8082 should be fine.